include/app/tabs/camera.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef CAMERA_HPP_
+#define CAMERA_HPP_
+
+#include <QMediaPlayer>
+#include <QVideoWidget>
+#include <QLabel>
+
+#include <ui_camera_settings.h>
+
+class CameraTab : public QWidget {
+    Q_OBJECT
+
+   public:
+    CameraTab(QWidget *parent = nullptr);
+
+   signals:
+    void media_status_changed( QString );

```

Comment:
no extra spaces around the param and should have a variable name

Reply:
Yeah, sorry about that. This is the coding style I use in my own projects and I missed this one when doing my conformity sweep.